discord/http.py

```diff
@@ -51,11 +51,13 @@
 import aiohttp
 
 from .errors import HTTPException, Forbidden, NotFound, LoginFailure, DiscordServerError, GatewayNotFound
+from .embeds import MAX_MESSAGE_EMBEDS
 from .gateway import DiscordClientWebSocketResponse
 from .file import File
 from .mentions import AllowedMentions
 from . import __version__, utils
 from .utils import MISSING
+print(__name__)
```

Comment:
Did you forget to remove this?

Reply:
@DJJ05 Sorry, yes, I forgot, I will fix it, one moment.

Reply:
Fixed

Reply:
👍